packages/flutter/lib/src/material/slider.dart

```diff
@@ -401,8 +402,10 @@ class Slider extends StatefulWidget {
   ///  * [MaterialState.hovered].
   ///  * [MaterialState.focused].
   ///  * [MaterialState.disabled].
+  /// {@endtemplate}
   ///
-  /// If this property is null, [MaterialStateMouseCursor.clickable] will be used.
+  /// If null, then the value of [SliderThemeData.mouseCursor] is used. If that
+  /// is also null, then [MaterialStateMouseCursor.clickable] is used.
```

Comment:
Even though it's implicitly mentioned above, it would be helpful to include a "see also" section that points at `MaterialStateMouseCursor`